### PR TITLE
[Feature] Update to Fluent 0.8.8

### PR DESCRIPTION
**Pull request by emlynmac**
## Purpose
Update the Fluent UI dependency to 0.8.8

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Pull code, build demo app and run; ensure no changes to UI

## What to Check
All UI functionality remains the same

### CODE REVIEW

**Review by palatter — DISMISSED** [review]
Looks good. Will test during standup.

**Comment by jpeng-ms** [comment]
thanks, one thing I noticed is that the drawer background colour seems to be inconsistent with cell background:

image
this applies to participant list and end call drawer as well

**Comment by emlynmac** [comment]
@jpeng-ms 
Looks like the internal layout of the cell isn't taking into account the height of the text. We might need to raise an issue with Fluent for the calculation of the height of the cell; it seems like this isn't dynamic here.

**Comment by emlynmac** [comment]
> thanks, would you mind to create mini-project that shows this is an issue on their end? I believe they would ask us to do this anyways for bug report

It's reproducible with the general way we're using it so I've created that demo app and raised the issue:
https://github.com/microsoft/fluentui-apple/issues/1351

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/FluentUI/Wrapper/CompositeParticipantsListCell.swift [hunk 1]
```diff
@@ -38,7 +38,7 @@ class CompositeParticipantsListCell: TableViewCell {
                                                          onHoldString: viewModel.getOnHoldString(),
                                                          isMuted: viewModel.isMuted)
         setup(title: viewModel.getCellDisplayName(with: participantViewData),
-              customView: avatarView,
+              customView: avatar,
```
Comment (jpeng-ms) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/FluentUI/Wrapper/CompositeParticipantsListCell.swift:41]: shouldn't this be ```avatar.view``` based on line 24 in the original file?

Comment (emlynmac) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/FluentUI/Wrapper/CompositeParticipantsListCell.swift:41]: Not anymore; part of the API changes.

**Review by Leah-Xia-Microsoft — DISMISSED** [review]

**Comment by jpeng-ms** [comment]
hi so I was testing on iPad, it seems the camera view would get cutoff when it goes to Split View. Do you know if this is due to Fluent UI or this is a bug in our app?
Simulator Screen Shot - iPad (9th generation) - 2022-11-10 at 11 03 43

Simulator Screen Shot - iPad (9th generation) - 2022-11-10 at 11 04 38



Simulator Screen Shot - iPad (9th generation) - 2022-11-10 at 11 03 50

**Comment by jpeng-ms** [comment]
also the on hold button doesn't seem to be fixed, I can still reproduce it:

IMG_0108

**Comment by emlynmac** [comment]
> was testing on iPad, it seems the camera view would get cutoff when it goes to Split View. Do you know if this is due to Fluent UI or this is a bug in our app?

Is this something that exists in the regular app, or only with this updated fluent UI version?

**Comment by emlynmac** [comment]
> also the on hold button doesn't seem to be fixed, I can still reproduce it:
> 
> IMG_0108

@jpeng-ms This issue seems to be due to `.fixedSize()` in `LobbyOverlayView.swift:45` pinning the size of the button incorrectly.

**Comment by jpeng-ms** [comment]
tbh I don't think it's an issue on our side though, It looks fine on our test flight build: 

IMG_0110

**Review by jpeng-ms — APPROVED** [review]
tested, looks good!

**Comment by jpeng-ms** [comment]
hi so I tested iPad running develop, I can't reproduce the video bug, so I guess there's something wrong with Fluent UI even though I am not sure if we used any components for the video view: 
image

**Review by palatter — APPROVED** [review]
Looks good. Tested on iPhone 14 pro max

**Comment by jpeng-ms** [comment]
hey was the Split View bug an issue with simulator? were you able to reproduce it on real devices?

**Pull request merged**